multihost_test/Bugzillas/test_bz_automation.py

```diff
@@ -225,3 +225,31 @@ def test_bz_2012929(self, multihost, create_backup):
             assert len(execute_cmd(multihost, f"grep -c container "
                                               f"/etc/{f_file}").stdout_text.split()) < 2
         execute_cmd(multihost, "userdel -rf container")
+
+    def test_bz_1994269(self, multihost):
+        """
+        :title: Stop allocating ID 65536 (reserved) for new users/groups
+        :id: b012f94e-d73d-11ed-b7e6-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1994269
+        :steps:
+          1. Manually manage the sub[ug]id ranges
+          2. Add User
+          3. Check /etc/subuid
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Useradd honors the predefined values in /etc/subuid,
+            /etc/subgid and doesn't add different values for the new created user.
+        """
+        execute_cmd(multihost, "cp -vf /etc/login.defs /etc/login.defs_bkp")
+        assert "UID_MAX" and "60000" in execute_cmd(multihost, "grep UID_ /etc/login.defs").stdout_text
+        execute_cmd(multihost, "useradd okuser")
+        assert "okuser" in execute_cmd(multihost, "id okuser").stdout_text
+        execute_cmd(multihost, "sed -i 's/UID_MAX                 "
+                               "60000/UID_MAX                 "
+                               "90000/g' /etc/login.defs")
```

Comment:
Can you replace all those white spaces by a regex? This would make the test easier to understand and easier to maintain.